src/current/releases/molt.md

```diff
@@ -18,6 +18,24 @@ To download the latest MOLT Fetch/Verify binary:
 
 {% include molt/molt-install.md %}
 
+## July 24, 2025
+
+MOLT Fetch/Verify 1.3.0 is [available](#installation).
+
+- Oracle is now supported on Fetch Docker images and standalone binaries.
+- Oracle is now supported on Replicator Docker images.
```

Comment:
It may be more clear to note that Oracle sources are now supported on Replicator Docker images and binaries